packages/ui5-application-inquirer/src/prompts/validators.ts

```diff
@@ -33,6 +33,10 @@ export async function validateFioriAppProjectFolder(targetDir: string): Promise<
     const appRoot = await findRootsForPath(targetDir);
     if (appRoot) {
         return t('validators.folderContainsFioriApp', { path: appRoot.appRoot });
+    } // Check targetDir is in CAP project sub directories && check if targetDir is a CAP project root directory
+    const isCapPath = await checkPathForCapApp(targetDir);
+    if (isCapPath) {
```

Comment:
nit-pick: Is there an actual need for `const isCapPath` or can this be simplified to 

```ts
if (await checkPathForCapApp(targetDir)) {
```

Reply:
Thank you, changed as requested.